addons/monitoring/dashboards/t1-gateways.json

```diff
@@ -1,1403 +0,0 @@
-{
```

Comment:
This dashboard should have never been here; it is one for the hosted env that is not a TSB one.